networkx/algorithms/bipartite/tests/test_extendability.py

```diff
@@ -29,6 +29,14 @@ def test_no_perfect_matching_raises():
         nx.bipartite.maximal_extendability(G)
 
 
+def test_not_strongly_connected_raises():
```

Comment:
It's probably worth updating this further to note that it's the connectivity of the *residual* graph, not the input graph, that is tested here. The residual graph is constructed internal to the function.

I don't have a sense for *why* a path graph results in a disconnected residual graph, so if there's a clear, concise explanation it's probably worth adding a comment!

```suggestion
def test_residual_graph_not_strongly_connected_raises():
```